shared/src/test/diff/mlscript/MutArray.mls

```diff
@@ -70,10 +70,8 @@ update0 0 ((mut 1,))
 update1 0 ((mut 1,))
 //│ = []
 
-:re // FIXME
 update2 0 ((mut 1,))
-//│ Runtime error:
-//│   TypeError: ((intermediate value) , []) is not a function
+//│ = []
```

Comment:
Why has this error gone?

Reply:
IDK... and I also don't know why this type checks, BTW! `((mut 1,))` is not supposed to be an array. Let me look into it...

Reply:
OH never mind, I missed that this was in the old parser. In the old parser, this is indeed an array.